src/bindist/autodos7.bat

```diff
@@ -0,0 +1,6 @@
+@ECHO Off
+PATH D:\OPENDOS;D:\EMUBIN;D:\
+VERIFY OFF
+PROMPT [OpenDOS] $P$G
+SET OPENDOSCFG=D:\OPENDOS
+CALL D:\EMUBIN\UXMACROS.BAT
```

Comment:
I just took a look at uxmacros.bat. It seems the Caldera engineers wanted to make life a bit easier for people working a lot with both unix and dos environments:
```
doskey ls="dir $1"
doskey cp="copy $1 $2"
doskey rm="del $1"
doskey mv="rename $1 $2"
doskey cat="type $1"
```
Having newlines at the end of batch files triggers DOS to put an extra C:\\> on the screen in my experience, so I tend to avoid them. It seems DOSes process empty lines.